### PR TITLE
create basic flask app

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

Added basic flask app with index, 404 and 500 templates

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1268

### CODE REVIEW

**Comment by nottrobin** [comment]
Looks great

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+PORT=8002
```
Comment (nottrobin) [inline .env]: This port should be unique, and should be reserved in the [ports table](https://github.com/canonical-web-and-design/practices/blob/master/project-structure/ports.md)

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+canonicalwebteam.flask-base==0.1.0
+canonicalwebteam.templatefinder==0.1.1
+Flask==1.0.2
+python-frontmatter==0.4.5
```
Comment (jpmartinspt) [inline requirements.txt]: @nottrobin Are this not installed by `canonicalwebteam.templatefinder==0.1.1`

Comment (nottrobin) [reply requirements.txt]: Yeah, @sowasred2012 please without this dependency

Comment (jpmartinspt) [reply requirements.txt]: Same for `mistune` and `bleach`.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+canonicalwebteam.flask-base==0.1.0
+Flask==1.0.2
+python-frontmatter==0.4.5
```
Comment (jpmartinspt) [inline requirements.txt]: Keep canonicalwebteam.templatefinder==0.1.1, remove bleach mistune and frontmatter.

**Pull request merged**